cmd/tree-walk.go

```diff
@@ -98,6 +98,8 @@ type listDirFunc func(bucket, prefixDir, prefixEntry string) (entries []string,
 // 4. XL backend multipart listing - isLeaf is true if the entry is a directory and contains uploads.json
 type isLeafFunc func(string, string) bool
 
+type isLeafDirFunc func(string, string) bool
```

Comment:
you need to explain this function just like `isLeafFunc`

Reply:
> you need to explain this function just like isLeafFunc

Done